test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/RelativePath.java

```diff
@@ -54,6 +54,16 @@ private static void doTest(String topArchiveName) throws Exception {
         int idx = appJar.lastIndexOf(File.separator);
         String jarName = appJar.substring(idx + 1);
         String jarDir = appJar.substring(0, idx);
+
+        // Create CDS Archive
+        dump(topArchiveName, "-Xlog:cds",
+            "-Xlog:cds+dynamic=debug",
+            "-cp", "." + File.separator + "hello.jar" + File.pathSeparator + appJar2,
+            "HelloMore")
+            .assertNormalExit(output-> {
+                    output.shouldContain("Written dynamic archive 0x");
+            });
+
```

Comment:
The new block also uses a relative path so it should be moved to under the `// relative path starting with "."` line.